flow/energy_models/power_demand.py

```diff
@@ -1,57 +1,127 @@
+"""Script containing the vehicle power demand model energy classes."""
```

Comment:
Just curious where did all these models come from?

Reply:
The physics-based model that we got from Benni:
![image](https://user-images.githubusercontent.com/7437172/86872526-683f6f00-c091-11ea-841c-03e9d50f9008.png)